build.zig

```diff
@@ -25,96 +28,114 @@ pub fn build(b: *Builder) !void {
 
     const fmt_step = b.addFmt(&[_][]const u8{
         "build.zig",
+        "rt.zig",
         "src",
         "test",
     });
     b.default_step.dependOn(&fmt_step.step);
 
+    comptime var available_tests: []const u8 = "";
```

Comment:
It doesn't look like this is ever used

Reply:
Ah, I thing this was the old description for the help, I shall remove